vector/fetch_vectors.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+while read hash; do
+  lines=$(find ./data -name "${hash}*" | wc -l)
+	if [ $lines -eq 0 ]; then
+    echo "Fetching $hash"
+    curl https://ipfs.io/ipfs/"$hash" -o tmp
```

Comment:
Can you make this script fail if curl fails

Reply:
good idea! done.